README.md

```diff
@@ -18,12 +18,12 @@ Install from [Hex.pm](https://hex.pm/packages/absinthe_relay):
 ```elixir
 def deps do
   [
-    {:absinthe_relay, "~> 1.5.0"}
+    {:absinthe_relay, "~> 1.6.0"}
   ]
 end
 ```
 
-Note: Absinthe.Relay requires Elixir 1.10 or higher.
+Note: Absinthe.Relay requires Elixir 1.11 or higher.
```

Comment:
Can you also change this in the `mix.exs` file?

Reply:
it's already in `mix.exs`

![image](https://github.com/absinthe-graphql/absinthe_relay/assets/16337910/48e2ecff-000d-4973-9e56-d078706328de)

Reply:
so maybe the version should be `1.5.3` then, not `1.6.0`?

Reply:
Yeah that is correct, if we are already requiring 1.11 in `mix.exs` and it's only a doc change and the warn -> warning change then this should be 1.5.3.

Reply:
done